spec/2021.12/API_specification/array_object.rst

```diff
@@ -170,7 +170,8 @@ A conforming implementation of the array API standard must provide and support a
     -   `operator.ne(x1, x2) <https://docs.python.org/3/library/operator.html#operator.ne>`_
     -   `operator.__ne__(x1, x2) <https://docs.python.org/3/library/operator.html#operator.__ne__>`_
 
-Comparison operators should be defined for arrays having any data type.
+:meth:`.array.__lt__`, :meth:`.array.__le__`, :meth:`.array.__gt__`, :meth:`.array.__ge__` are only defined for arrays having real-valued data types. Other comparison operators should be defined for arrays having any data type.
```

Comment:
This change should not be backported to the `2021.12` revision, as complex-valued dtypes were not part of the standard.

Reply:
Resolved in https://github.com/data-apis/array-api/pull/736/commits/0b5e90d20d578db70809781c0bdb9ec5786783d4.